### PR TITLE
New assignments

### PR DESCRIPTION
**Pull request by jhmigueles**
New assignment solutions and fixed name of ID in sleeplog

### CODE REVIEW

File: Assignment1_solution.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+# 1 - load GGIR
+library(GGIR)
+
+# 2 - GGIR function call
+
+GGIR(datadir = "C:/GGIRtraining/file/",
+     outputdir = "C:/GGIRtraining/assignments/",
+     mode = 1:2)
```
Comment (vincentvanhees) [inline Assignment1_solution.R]: Add:
do.report = 2
visualreport = FALSE

File: Assignment3_solution.R [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+# 1 - load GGIR
+library(GGIR)
+
+# 2 - GGIR function call
+
+GGIR(datadir = "C:/GGIRtraining/file/",
+     outputdir = "C:/GGIRtraining/assignments/",
+     mode = 3:4,
```
Comment (vincentvanhees) [inline Assignment3_solution.R]: This assumes that the participant is still working with the same output folder as the day 1.
Should we add that as a comment here?

For example, `mode = 3:4, # we assume that the milestone data from GGIR part 1 and 2 are still in output directory`

File: Assignment3_solution.R [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+     mvpathreshold = c(100, 140),
+     # advanced
+     qwindow = c(0, 7, 22, 24),
+     # assignment 3.1
+     HASIB.algo = "vanHees2015",
+     HASPT.algo = "HDCZA"
+     )
+
+# 2 - copy results/part4_nightsummary_clean.csv
+file.copy(from = "C:/GGIRtraining/assignment 2/output_file/results/part4_nightsummary_sleep_cleaned.csv",
```
Comment (vincentvanhees) [inline Assignment3_solution.R:44]: Comment that user can also do this manually outside R? We do not want people to think that R requires all file management to happen via R code.

File: Assignment4_solution.R [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+library(GGIR)
+
+# 2 - GGIR part5
+GGIR(datadir = "C:/GGIRtraining/file/",
+     outputdir = "C:/GGIRtraining/assignments/",
+     desiredtz = "Europe/Madrid",
+     do.enmo = TRUE,   # optional (default)
+     do.anglez = TRUE, # optional (default)
+     strategy = 1,
+     hrs.del.start = 1, hrs.del.end = 1,
```
Comment (vincentvanhees) [inline Assignment4_solution.R:24]: Should we add comments that the user can leave all arguments related to part 1, 2, 3, and 4, but that those are not used by part 5.

File: Assignment1_solution.R (hunk 1 above)
Comment (jhmigueles) [reply Assignment1_solution.R]: Should we specify that attendees should turn off the report generation for parts 4 and 5 in the slide for the assignment?

File: Assignment3_solution.R (hunk 2 above)
Comment (jhmigueles) [reply Assignment3_solution.R]: Done, and this is also specified in the slide for the assignment

File: Assignment3_solution.R (hunk 3 above)
Comment (jhmigueles) [reply Assignment3_solution.R:44]: Done

File: Assignment4_solution.R (hunk 4 above)
Comment (jhmigueles) [reply Assignment4_solution.R:24]: Comments in all assignments are improved. Now the assignments are built on the previous one, which is more realistic to how a project is developed. Slides also clarify this now.

**Pull request merged**